server/migrations/20190312121718-create-recipe.js

```diff
@@ -0,0 +1,46 @@
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    return queryInterface.createTable('Recipes', {
+      id: {
+        allowNull: false,
+        autoIncrement: true,
+        primaryKey: true,
+        type: Sequelize.INTEGER
+      },
+      title: {
+        type: Sequelize.STRING
+      },
+      slug: {
+        type: Sequelize.STRING
+      },
+      ingredients: {
```

Comment:
**[question]** Will we be storing the ingredients and their amounts together?

Reply:
Yes. We will store the ingredients and the amounts together. An example will be, `1/2 cups of rice`. The ingredients will be an array of strings.